### ISSUE TITLE
donn't work amministrative mode

### ISSUE THREAD

**Issue by PacoRaban58** [NexovaDev/UpdateHub#164]
### Debug Info:
```
App version: 2.1.0
Chrome version: 114.0.5735.243
Node version: 18.15.0
Electron version: 25.3.1
OS version: 22631.2191
Device ID: a8f984a4-c4fc-4b00-9239-7dafb4d78aa0
```

**Comment by PacoRaban58** [comment]
for amministrave mode is impossible to use

**Comment by KpochMX** [comment]
whats the point of this app if i need to launch it twice with admin right and with out and i need to click UPDATE on each item.........
the best for me is
Open CMD and type "Winget update -h --all" this will update everything with any other interactionalso can set up as a scheduled task

**Comment by KK-Designs** [comment]
This is fixed as of [v2.3.1](https://github.com/NexovaDev/UpdateHub/releases/tag/v2.3.1).
